boinc-client/start.sh

```diff
@@ -11,14 +20,18 @@ if [[ -z $ACCOUNT_KEY ]]; then
   echo 'Account key undefined - using balena key'
 else
   echo 'Account key set'
-  sed -i -e 's|<authenticator>[0-9a-z_]\{1,\}</authenticator>|<authenticator>'"$ACCOUNT_KEY"'</authenticator>|g' /usr/app/boinc/account_boinc.bakerlab.org_rosetta.xml
+  sed -i -e 's|<authenticator>[0-9a-z_]\{1,\}</authenticator>|<authenticator>'"$ACCOUNT_KEY"'</authenticator>|g' account_boinc.bakerlab.org_rosetta.xml
+fi
+
+if [[ -z $SKIP_BOINC_MEM_SETTINGS_CHECK ]]; then
+  validate_ram_settings
 fi
 
 totalmem=$(awk '/^MemTotal:/{print $2}' /proc/meminfo)
 
-if [ "$totalmem" -lt "1500000" ]; then
+if [[ -z $SKIP_BOINC_CPU_SETTINGS_CHECK && "$totalmem" -lt "1500000" ]]; then
```

Comment:
As per your feedback in the issue and my tests on the Pi 4 2GB just now, let's put this back to the 2.5GB value it was at previously.

```suggestion
if [[ -z $SKIP_BOINC_CPU_SETTINGS_CHECK && "$totalmem" -lt "2500000" ]]; then
```